Node.gitignore

```diff
@@ -70,7 +70,13 @@ typings/
 .vuepress/dist
 
 # Serverless directories
-.serverless
+.serverless/
 
 # FuseBox cache
 .fusebox/
+
+#DynamoDB Local files 
+.dynamodb/ 
```

Comment:
Any links to StackOverflow discussions or other documentation about this change?

Reply:
If you have an offline dynamodb instance to test your node app , it will store the offline test db data in that folder that you won't need on your deployment. 
Frequently used when deploying serverless node js apps.  

From AWS documentation : 
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html 

The NPM packages : 
https://www.npmjs.com/package/serverless-dynamodb-local
https://www.npmjs.com/package/dynamodb-local